scio-tensorflow/src/main/scala/com/spotify/scio/tensorflow/TensorFlowSCollectionFunctions.scala

```diff
@@ -116,14 +116,14 @@ class TensorFlowSCollectionFunctions[T: ClassTag](@transient val self: SCollecti
    *
    * @param graphUri URI of pre-trained/saved TensorFlow model
    * @param fetchOps names of [[org.tensorflow.Operation]]s to fetch the results from
-   * @param config configuration parameters for the session specified as a serialized
-   *               `org.tensorflow.framework.ConfigProto` protocol buffer.
-   * @param inFn translates input elements of T to map of input-operation ->
-   *             [[org.tensorflow.Tensor Tensor]]. This method takes ownership of the
-   *             [[org.tensorflow.Tensor Tensor]]s.
-   * @param outFn translates output of prediction from map of output-operation ->
-   *              [[org.tensorflow.Tensor Tensor]], to elements of V. This method takes ownership
-   *              of the [[org.tensorflow.Tensor Tensor]]s.
+   * @param config   configuration parameters for the session specified as a serialized
```

Comment:
We don't normally align doc. It doesn't affect generated html and just makes edits harder.

Reply:
👍 fixed my intellij for next time

Reply:
Undo the indent changes here?